libraries/skeletons.md

```diff
@@ -32,3 +32,7 @@ D3.js lib.
 
 ### [Udash Generator](http://guide.udash.io/#/bootstrapping/generators)
 Project generator for the Udash framework - creates a customized, complete and runnable Udash based application.
+
+### [Develop WeChat application with Scala.js](https://github.com/kaiwu/weui-scalajs)
+The project implements [WeUI demo](https://github.com/Tencent/weui-wxss/) entirely with Scala.js and sbt-less,
+showing that Scala.js is a solid development platform for [WeChat mini-programs aka WeApp](https://en.wikipedia.org/wiki/WeChat).
```

Comment:
The second sentence is misleading by contrast to the communication we do about Scala.js elsewhere, i.e., that fact that we never say "Scala.js is good for that specific thing". Indeed, our form of communication is more like "Scala.js can do everything that JS can do, so you can use it anywhere JS is at play." If we reinforce the "Scala.js is solid for WeApp" but do not reinforce that kind of message for other platforms, by contrast, we are implicitly saying that Scala.js is not as solid for the other platforms.

I would suggest an entirely different wording to match the other skeleton descriptions better:
```markdown
### [WeUI demo with Scala.js](https://github.com/kaiwu/weui-scalajs)
Skeleton of a [WeChat mini-program aka WeApp](https://en.wikipedia.org/wiki/WeChat) with Scala.js and sbt-less: a port of the [WeUI demo](https://github.com/Tencent/weui-wxss/).
```